documentation/Not_supported_Cygwin.adoc

```diff
@@ -0,0 +1,21 @@
+:toc:
+toc::[]
+
+= Not supported: Cygwin
```

Comment:
For our automated AsciiDoc processing we must not use special characters like colon in section titles:
https://github.com/devonfw/docgen/blob/master/documentation/guidelines.asciidoc

could you also rename this file to just `cygwin.adoc`.
Over time our support for cygwin may increase one day and then we do not need to rename the file, etc.

After merging, I can add some hints here how cygwin user can make it still work.